internal/eventstore/handler/handler_projection_test.go

```diff
@@ -846,14 +846,20 @@ func testUpdate(t *testing.T, expectedStmtCount int, returnedErr error) Update {
 	}
 }
 
-func testReduce(stmts ...Statement) Reduce {
-	return func(event eventstore.EventReader) ([]Statement, error) {
-		return stmts, nil
+func testReduce(stmts Statement) Reduce {
+	return func(event eventstore.EventReader) (*Statement, error) {
+		return &stmts, nil
+	}
+}
+
+func testReduceEmpty() Reduce {
```

Comment:
IMO a reduce should at least end in a `NoOpStatement` if no error occurred.

Reply:
this was done first, because the previous version had testReduce which had a variadic param

i remove the function and always provide a newTestStatement (equivalent to noopstatement)